warehouse/packaging/models.py

```diff
@@ -683,6 +684,8 @@ def __table_args__(cls):  # noqa
     # of all of them and then remove this column.
     allow_multiple_sdist = Column(Boolean, nullable=False, server_default=sql.false())
 
+    archived = Column(Boolean, nullable=False, server_default=sql.false())
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏it might be nice to start adding `comment="..."` to. columns when possible, as they serve for good inline documentation, and ought to be added to the migration and be a comment in the db as well.
https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.Column.params.comment

Reply:
If you figure out some way to enforce that all new columns do via linting, I think it'd be a great "good first PR" for someone to take on backfilling!

Reply:
https://github.com/pypi/warehouse/pull/13397/commits/afaf7f680658663a21106042c8b6265a2b467876

Reply:
https://pypi.org/project/flake8-sqlalchemy/ - let me know what you think - it's super alpha.